src/foam/comics/v2/DAOBrowseControllerView.js

```diff
@@ -116,12 +117,26 @@ foam.CLASS({
       },
       code: function() {
         if ( ! this.stack ) return;
-        this.stack.push({
-          class: 'foam.comics.v2.DAOCreateView',
-          data: ((this.config.factory && this.config.factory$cls) ||  this.data.of).create({ mode: 'create'}, this),
-          config$: this.config$,
-          of: this.data.of
-        }, this.__subContext__);
+
+        if ( this.config.DAOCreateView ){
+          this.stack.push({
+            class: 'foam.comics.v2.DAOCreateView',
+            data: ((this.config.factory && this.config.factory$cls) ||  this.data.of).create({ mode: 'create'}, this),
+            config$: this.config$,
+            of: this.data.of
+          }, this.__subContext__);
+        } else {
+          this.stack.push({
+            class: this.config.wizardDetailView.class,
```

Comment:
Why do we need this? Couldn't we just set the config.DAOCreateView to this?

Reply:
this is an option if the user have create a custom view.